test/fixtures/imminence/national.json

```diff
@@ -0,0 +1,51 @@
+{

```

Comment:
I believe this file can be deleted once you switch to using `imminence_has_areas_for_postcode`.